tests/test_run_docker.sh

```diff
@@ -4,11 +4,11 @@
 
 USER_KEY=$1
 SUBSCRIPTION_ID=$2
-ENV=$3
+API_HOST=$3
```

Comment:
I chose this because this is public facing and I didn't want to include our staging url if it isn't necessary. I don't imagine a customer would use staging here. This is what I did with the python streaming client as well.

Reply:
true, good point